rxnetty-http/src/test/java/io/reactivex/netty/protocol/http/client/HttpClientTest.java

```diff
@@ -272,6 +274,101 @@ public void call() {
 
     }
 
+    @Test(timeout = 60000)
+    public void testRequestWithNoContentLengthHeaderOrContentReturnsEmptyBody() {
+        serverRule.startServer();
+
+        serverRule.assertRequestEquals(
+            new Func1<HttpClient<ByteBuf, ByteBuf>, Observable<HttpClientResponse<ByteBuf>>>() {
+                @Override
+                public Observable<HttpClientResponse<ByteBuf>> call(HttpClient<ByteBuf, ByteBuf> client) {
+                    return client.createGet("/");
+                }
+            },
+            "GET / HTTP/1.1\r\n" +
```

Comment:
I didn't see the code in `HttpClientRule` for this the first time round (I started from the server rule). I've push a commit to address this that can be squashed in if it's good.